Tools/CISupport/ews-build/steps.py

```diff
@@ -258,13 +258,15 @@ def get_number_of_prs_with_label(self, label):
         query_body = '{repository(owner:"%s", name:"%s") { pullRequests(labels: "%s") { totalCount } } }' % (owner, name, label)
         query = {'query': query_body}
         response = yield self.query_graph_ql(query)
-        if response:
-            num_prs = response['data']['repository']['pullRequests']['totalCount']
-            yield self._addToLog('stdio', 'There are {} PR(s) in safe-merge-queue.\n'.format(num_prs))
-            defer.returnValue(num_prs)
-        else:
+        if not response:
             yield self._addToLog('stdio', 'Failed to retrieve number of PRs.\n')
-            defer.returnValue(None)
+            return defer.returnValue(None)
+        elif 'errors' in response:
+            yield self._addToLog('stdio', response['errors'][0]['message'])
```

Comment:
`response['errors'][0]['message']` is prone to exception. won't be nice to have an exception while trying to print the error.